projects/Mallard/src/components/article/article-header/review-header.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import { metrics } from 'src/theme/spacing'
+import { ArticleImage } from '../article-image'
+import { getNavigationPosition } from 'src/helpers/positions'
+import { newsHeaderStyles } from '../styles'
+import { ArticleStandfirst } from '../article-standfirst'
+import { ArticleHeaderProps } from './types'
+import { Multiline } from '../../multiline'
+import { ArticleHeadline } from '../article-headline'
+import Quote from 'src/components/icons/Quote'
+import { getFont } from 'src/theme/typography'
+import { color } from 'src/theme/color'
+import { useArticle } from 'src/hooks/use-article'
+import { getFader } from 'src/components/layout/animators/fader'
+import { ArticleByline } from '../article-byline'
+import { ArticleKicker } from '../article-kicker'
+
+const ArticleFader = getFader('article')
+
+const styles = StyleSheet.create({
+    background: {
+        backgroundColor: color.palette.opinion.faded,
+    },
+    byline: {
+        fontFamily: getFont('titlepiece', 1).fontFamily,
+        width: '100%',
+    },
+})
+
+const style = StyleSheet.create({
+    sides: { paddingHorizontal: metrics.articleSides },
+    headline: { paddingBottom: metrics.vertical * 4 },
+    standfirst: { paddingBottom: metrics.vertical * 1 },
+})
+
+const ReviewHeader = ({
+    byline,
+    kicker,
+    headline,
+    image,
+    standfirst,
+}: ArticleHeaderProps) => {
+    const [color] = useArticle()
+    const navigationPosition = getNavigationPosition('article')
+    return (
+        <View style={[{ backgroundColor: color.faded }]}>
+            <View style={style.sides}>
+                {image ? (
+                    <ArticleFader buildOrder={1}>
```

Comment:
I wonder if there's any way we can abstract this build order thing out into it's own component so that we don't have to worry about adding it again for each header - something like `ArticleHeaderLayout`. Or are they different for each header?

Reply:
they are different for each header! we could be cheeky and use some magical global js to identify the build order at run time tho?

Reply:
No, that's fine - if they're different from each then that's cool 🆗